todo_app/mongo_repository.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from pprint import pprint
+
+import pymongo
+from bson import ObjectId
+from pymongo import ReturnDocument
+
+from todo_app.flask_config import Config
+from todo_app.item import Status, Item
+
+
+class MongoRepository:
+    def __init__(self):
+        config = Config()
+        pprint(f'db connection --> ${config.MONGO_DB_NAME}')
+        mongo_client = pymongo.MongoClient(config.MONGO_URL)
+        self.db = mongo_client[config.MONGO_DB_NAME]
+        self.todo_collection = self.db[config.MONGO_LIST_COLLECTION]
+
+    def add_item(self, tittle, description):
+        list_item = {
+            'name': tittle,
+            'desc': description,
+            'status': Status.NOT_STARTED.name,
+            'dateLastActivity': datetime.now(),
+        }
+        saved_list = self.todo_collection.insert_one(list_item)
+        pprint(f'Saved item with id = ${saved_list.inserted_id} to list collection')
+
+    def get_items(self):
+        items = []
+        for doc in self.todo_collection.find():
+            items.append(Item.from_response(doc))
+
+        return items
+
+    def move_to_done(self, item_id):
+        saved = self.todo_collection.find_one_and_update({'_id': ObjectId(item_id)},
+                                                         {'$set': {'status': 'COMPLETED'}},
+                                                         return_document=ReturnDocument.AFTER)
```

Comment:
You don't do anything with the returned document

Reply:
removed